README.md

```diff
@@ -13,7 +13,7 @@ Framework agnostic [content negotiation](https://tools.ietf.org/html/rfc7231#sec
 dependencies:
   athena-negotiation:
     github: athena-framework/negotiation
-    version: ~> 1.0.0
+    version: ~> 0.1.1
```

Comment:
```suggestion
    version: ~> 0.1.0
```

Can just use `0.1.0` here as `0.1.1` is mainly just documentation updates versus a critical bug/regression.